docs/analyzer.md

```diff
@@ -0,0 +1,24 @@
+# Analyzer
+
+FakeItEasy provides a Roslyn analyzer to detect incorrect usages of the library
+that cannot be prevented by the API or the compiler but will typically result in
+bugs at runtime.

```

Comment:
Are you worried that this forces linebreaks?
I'm seeing different wrapping on readthedocs.

![image](https://cloud.githubusercontent.com/assets/3275797/20282035/feb1dc3c-aa80-11e6-992d-746b3db5687c.png)